consensus-types/blocks/roblob.go

```diff
@@ -62,3 +62,31 @@ func (b *ROBlob) BodyRoot() [32]byte {
 func (b *ROBlob) ProposerIndex() primitives.ValidatorIndex {
 	return b.SignedBlockHeader.Header.ProposerIndex
 }
+
+// BlockRootSlice returns the block root as a byte slice. This is often more conveninent/concise
+// than setting a tmp var to BlockRoot(), just so that it can be sliced.
+func (b *ROBlob) BlockRootSlice() []byte {
```

Comment:
I am a little concerned with the possible usages of this function since there may be a data race when the root is being set.

Reply:
The root is *only* set in the constructor.